app/controllers/bookmarks_controller.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+class BookmarksController < CatalogController
+  include Blacklight::Bookmarks
+
+  # Overidden from Blacklight to add our tabbed interface
+  def index
+    @bookmarks = token_or_current_or_guest_user.bookmarks.where(record_type: 'catalog')
```

Comment:
🤔 maybe we can inject a default scope to blacklight's `Bookmark` model and prevent us from having to override this action.